internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer.go

```diff
@@ -96,12 +96,9 @@ func (*Sequencer) Initialize(r runtime.Runtime) []runtime.Phase {
 			r.State().Machine().Installed(),
 			"unmountBoot",
 			UnmountBootPartition,
-		).Append(
-			"resetNetwork",
-			ResetNetwork,
 		).Append(
 			"setupNetwork",
-			SetupDiscoveryNetwork,
+			SetupNetwork,
```

Comment:
merged two phases into one, so that sequence is the following:

1. `networkd.New()` lists interfaces and filters out those which are not operationalUp.
2. `Reset()` removes addresses and brings down the interfaces.
3. `Configure()` performs network configuration based on configuration (and brings up interfaces we need)